server/redis-credentials.ts

```diff
@@ -0,0 +1,18 @@
+import env from "./env";
+import pickBy from "lodash.pickby";
+
+const credentials = {
+  host: env.REDIS_HOST,
+  port: env.REDIS_PORT,
+  db: env.REDIS_DB,
+  ...(env.REDIS_PASSWORD && { password: env.REDIS_PASSWORD })
```

Comment:
I think not needed, just add like rest of fields and pickCredentials will filter it out if it didn't exist

Reply:
I don't think we need to filter credentials. I know you aimed at exact behaviour but db is a standard redis configuration which defaults to 0. Let's simplify the code by requiring all redis config with no filters.